internal/apiserviceref/apiserviceref.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 the Pinniped contributors. All Rights Reserved.
```

Comment:
If it were me, I would write a unit test here...

Reply:
I exercised it directly in an integration test which I found more valuable.